stdlib/scripts/run-tests.sh

```diff
@@ -23,6 +23,12 @@ mkdir -p "${BUILD_DIR}"
 
 source "${SCRIPT_DIR}"/build-stdlib.sh
 
+echo "Validating API documentation strings in the Standard Library."
```

Comment:
Same here, testing the docstrings is more something to do when a PR is nearly done. Something like a pre-commit. 
Contributors might want to run tests without checking the docstrings first.

Reply:
I agree this is more akin to `mojo format` sort of thing - something needed to be fixed up prior to landing a PR.  After this lands, do you have any interest in exploring hooking this up with the `pre-commit` things you've set up recently?  I think that would be great.